backend/nodes/image_util_nodes.py

```diff
@@ -326,7 +350,7 @@ def run(
     ) -> np.ndarray:
         """Adjusts the position of an image"""
 
-        h, w, _ = get_h_w_c(img)
```

Comment:
I think it's more explicit to destructure the tuple fully instead of doing `[:2]`.